src/test/java/com/liveramp/cascading_ext/combiner/lib/TestHyperLogLogPlusBackwardsCompatibility.java

```diff
@@ -0,0 +1,29 @@
+package com.liveramp.cascading_ext.combiner.lib;
+
+import com.clearspring.analytics.stream.cardinality.HyperLogLogPlus;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.*;
+
+public class TestHyperLogLogPlusBackwardsCompatibility {
+
+  @Test
+  public void testBackwardsCompatibility() throws IOException {
+    // verifies that we can read HLL sketches generated previously and hardcoded as resources
+
+    byte[] bytes = readHllBytes("hllp_small.bin");
+    HyperLogLogPlus hll = HyperLogLogPlus.Builder.build(bytes);
+    Assert.assertEquals(17815, hll.cardinality());
```

Comment:
What else do we do with the HLLs? Could we exercise all that as well if it's easy?

Reply:
Yup, added a test for the merge operation.